### PR TITLE
reword some mentions of Azure and GCP around MST

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Reworded some mentions of Azure and GCP to find where they are mentioned in relation to MST. Also removed MST cloud regions page.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: _partials/_cloud-mst-comparison.md [hunk 1]
```diff
@@ -2,11 +2,10 @@
...
   header="Timescale or Managed Service for TimescaleDB?"
 >
-If your workload lives in AWS, use [Timescale](/getting-started/latest/). Built
-and exclusively operated by Timescale, Timescale is a PostgreSQL data platform
-for time-series, events and analytics.
 
-If you need Azure or GCP, contact
-[Timescale support](https://www.timescale.com/contact/) about
-[Managed Service for TimescaleDB](/mst/latest/) instead.
+Looking for Timescale documentation? Built and exclusively operated by
```
Comment (Rajakavitha1) [inline _partials/_cloud-mst-comparison.md:6]:
```suggestion
Looking for instructions to start using Timescale? Built and exclusively operated by
```

File: about/timescaledb-editions.md [hunk 2]
```diff
@@ -60,11 +60,8 @@ to repair" and make modifications to the source code and run it in their own
 on-premises or cloud infrastructure. However, you cannot make modifications to
 the TimescaleDB Community Edition source code and offer it as a service.
 
-Timescale provides hosted versions of TimescaleDB Community Edition:
-[Timescale][timescale-cloud] and
-[Managed Service for TimescaleDB][mst]. Timescale is a cloud-native
-platform for time-series, hosted in AWS. MST is a database-as-a-service offering
-for TimescaleDB. MST is offered in AWS, Azure, and Google Cloud.
+You can access a hosted version of TimescaleDB Community Edition through
+[Timescale][timescale-cloud], which is a cloud-native platform for time-series.
```
Comment (Rajakavitha1) [inline about/timescaledb-editions.md:64]: I am not sure. I always thought that Timescale Cloud was not the hosted version of the TimescaleDB community.

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!!
Just a minor rewording about not using Timescale Documentation because there are already looking at Timescale Documentation.

File: about/timescaledb-editions.md (hunk 2 above)
Comment (Loquacity) [reply about/timescaledb-editions.md:64]: The original seemed to say both Cloud and MST were hosted versions of Community. Maybe @mfreed can confirm?

Comment (mfreed) [reply about/timescaledb-editions.md:64]: This is correct

**Pull request merged**